ts/components/Lightbox.tsx

```diff
@@ -111,11 +214,28 @@ export class Lightbox extends React.Component<Props, {}> {
   };
 
   private onKeyUp = (event: KeyboardEvent) => {
-    if (event.key !== 'Escape') {
-      return;
-    }
+    const { onClose } = this;
+    const { onNext, onPrevious } = this.props;
+    switch (event.key) {
+      case 'Escape':
+        onClose();
+        break;
 
-    this.onClose();
+      case 'ArrowLeft':
+        if (onPrevious) {
+          onPrevious();
+        }
+        break;
+
+      case 'ArrowRight':
```

Comment:
Gotta love the normalized React synthetic events 😄

Reply:
Yup. It would be an even bigger win if we were supporting multiple browsers!